ignite/cmd/scaffold_list.go

```diff
@@ -36,7 +36,7 @@ provides the logic to create, read, update, and delete instances of the type.
 For example, let's review a command that generates the code to handle a list of
 posts and each post has "title" and "body" fields:
 
-	ignite scaffold list post title body
+	ignite scaffold list post title body:string
```

Comment:
I'm not sure if the default `string` type should be explicit 🤔, explicit types are explained two paragraphs below this line. In theory users should specify only when the type is not the default. Arguably, it also makes commands and help text cleaner.

Reply:
I agree. Let's remove `:string`, otherwise users will think there is a difference between `title` (implicit `string`) and `body:string` (explicit `string`).

Reply:
I agree here, this looks very weird to see those explicit `: string` and looks mandatory otherwise